source/numeric/ConstantValue.cpp

```diff
@@ -37,51 +37,54 @@ std::string ConstantValue::toString() const {
                 return "$"s;
             else if constexpr (std::is_same_v<T, Elements>) {
                 FormatBuffer buffer;
-                buffer.append("[");
+                buffer.append("'{");
                 for (auto& element : arg) {
-                    buffer.append(element.toString());
+                    buffer.append(element.toString(abbreviateThresholdBits));
```

Comment:
Did you want to pass the exactUnknowns flag down into these other toString() calls as well?

Reply:
Yes, I did. Thanks for catching that; I propagated it.